### ISSUE TITLE
Do not track Timber logs

### ISSUE THREAD

**Issue by ThomasArtProcessors** [getsentry/sentry-java#2468]
### Problem Statement

How can I remove the tracking of Timber logs?
Our usage of Timber is for logs only, not crashes. I updated Sentry and am now receiving a plethora of "crash" logs.
The strange thing is that I'm only "planting" Timber for DEBUG builds, yet I'm seeing all these reports in our production release.
I couldn't find a way to simply NOT report Timber logs, beyond setting 
```
    autoInstallation {
        enabled = false
    }
    ```
    But that comes at the cost of setting everything up manually.
    
    TLDR: Can we simple turn off reporting on TImber logs ( and how are they still reporting when Timber is only planted in DEBUG builds? )

**Comment by romtsn** [comment]
hi, sorry for the inconvenience. The logs are still being reported because we automatically detect if you have a timber dependency on your app and then install the TimberIntegration automatically. You can keep autoInstallation enabled, but remove the integration at runtime via [manual init](https://docs.sentry.io/platforms/android/configuration/manual-init/):

```kotlin
SentryAndroid.init(context) { options ->
    options.integrations.removeAll { it is SentryTimberIntegration }
}
```

This is still not the most optimal way, so I've filed https://github.com/getsentry/sentry-android-gradle-plugin/issues/426 to better facilitate the configuration of the plugin. Let us know if you still have questions!

**Comment by ThomasArtProcessors** [comment]
I applied this manual fix but Sentry is still capturing the log from all my external libraries / dependencies.
This has caused extreme noise and an increase in our quota which is now forcing us to pay extra money for it...
I am now forced to remove Sentry completely in the meantime.
Is there a better solution for this?
Thanks.

**Comment by romtsn** [comment]
hi @ThomasArtProcessors, apologies for not coming back earlier. I tested my proposed solution and it indeed does not work, sorry for that! 

I think the best way to disable the Timber integration for now would be to completely remove the `sentry-android-timber` dependency from your configurations in `app/buid.gradle` like this:

```gradle
configurations.configureEach {
    exclude group: "io.sentry", module: "sentry-android-timber"
}
```

or if you're using `.kts`:

```kotlin
configurations.configureEach {
    exclude(group = "io.sentry", module = "sentry-android-timber")
}
```

until we implement the proper solution on the gradle plugin side. 

cc @souredoutlook might help you with the quota overload. Nick, would it be possible to increase the event quota for @ThomasArtProcessors, since it's kind of our fault that we do not provide an easy way to disable the messages/events here. I'll add some docs around that meanwhile. Thank you all
